doc/string.rdoc

```diff
@@ -49,34 +49,138 @@ class String
   end
 
   #  call-seq:
-  #    split(separator = $;, limit = nil) -> array
-  #    split(separator = $;, limit = nil) {|substring| ... } -> self
+  #    bytes -> array_of_bytes
+  #
+  #  Returns an array of the bytes in +self+:
+  #
+  #    'hello'.bytes # => [104, 101, 108, 108, 111]
+  #    'тест'.bytes  # => [209, 130, 208, 181, 209, 129, 209, 130]
+  #    'こんにちは'.bytes
+  #    # => [227, 129, 147, 227, 130, 147, 227, 129, 171, 227, 129, 161, 227, 129, 175]
+  #
+  def bytes; end
+
+  #  call-seq:
+  #    each_byte {|byte| ... } -> self
+  #    each_byte               -> enumerator
+  #
+  #  Calls the given block with each successive byte from +self+;
+  #  returns +self+:
+  #
+  #    'hello'.each_byte {|byte| print byte, ' ' }
+  #    print "\n"
+  #    'тест'.each_byte {|byte| print byte, ' ' }
+  #    print "\n"
+  #    'こんにちは'.each_byte {|byte| print byte, ' ' }
+  #    print "\n"
+  #
+  #  Output:
+  #
+  #    104 101 108 108 111
+  #    209 130 208 181 209 129 209 130
+  #    227 129 147 227 130 147 227 129 171 227 129 161 227 129 175
+  #
+  #  Returns an enumerator if no block is given.
+  def each_byte; end
+
+
+  #  call-seq:
+  #    each_line(line_sep = $/, chomp: false) {|substring| ... } -> self
+  #    each_line(line_sep = $/, chomp: false)                    -> enumerator
+  #
+  #  With a block given, forms the substrings ("lines")
+  #
+  #  that are the result of splitting +self+
+  #  at each occurrence of the given line separator +line_sep+;
+  #  passes each line to the block;
+  #  returns +self+:
+  #
+  #    s = <<~EOT
+  #    This is the first line.
+  #    This is line two.
+  #
+  #    This is line four.
+  #    This is line five.
+  #    EOT
+  #
+  #    s.each_line {|line| p line }
+  #
+  #  Output:
+  #
+  #    "This is the first line.\n"
+  #    "This is line two.\n"
+  #    "\n"
+  #    "This is line four.\n"
+  #    "This is line five.\n"
+  #
+  #  With a different +line_sep+:
+  #
+  #    s.each_line(' is ') {|line| p line }
+  #
+  #  Output:
+  #
+  #    "This is "
+  #    "the first line.\nThis is "
+  #    "line two.\n\nThis is "
+  #    "line four.\nThis is "
+  #    "line five.\n"
+  #
+  #  With +chomp+ as +true+, removes the trailing +line_sep+ from each line:
+  #
+  #    s.each_line(chomp: true) {|line| p line }
+  #
+  #  Output:
+  #
+  #    "This is the first line."
+  #    "This is line two."
+  #    ""
+  #    "This is line four."
+  #    "This is line five."
+  #
+  #  With an empty string as +line_sep+,
+  #  forms and passes "paragraphs" by splitting at each occurrence
+  #  of two or more +line_sep+:
```

Comment:
`line_sep` is the empty string, I think you mean newlines

```suggestion
  #  of two or more newlines:
```

Reply:
Fixed.